### PR TITLE
Debuginfo check implementation

### PR DESCRIPTION
**Pull request by Gabib32**
Add two checks for the debuginfo. The first determines
whether the debug package contains files at all, and the
second determines whether the package is considered complete.

### CODE REVIEW

**Comment by Gabib32** [comment]
This PR is only for commit 0f98112

File: cmd/debuginfo.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+func init() {
+	debuginfoCmd.Flags().StringVarP(&debuginfoFlags.mixName, "name", "n", "clear", "name of data group")
+	debuginfoCmd.Flags().StringVarP(&debuginfoFlags.version, "version", "v", "0", "version to check")
+	debuginfoCmd.Flags().BoolVar(&debuginfoFlags.latest, "latest", false, "get the latest version from upstreamURL")
+	debuginfoCmd.Flags().BoolVar(&debuginfoFlags.verbose, "verbose", false, "lists failed package results")
+}
+
+var debuginfoCmd = &cobra.Command{
+	// TODO
```
Comment (matthewrsj) [inline cmd/debuginfo.go]: todo?

File: cmd/debuginfo.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+			}
+			if strings.HasPrefix(fname, "/usr/lib/debug") {
+				libFlag = true
+			}
+			// no need to read all files, if both lib and src files found
+			if srcFlag && libFlag {
+				break
+			}
+		}
+		if !srcFlag {
```
Comment (matthewrsj) [inline cmd/debuginfo.go]: what about `if !libFlag`? You could loop over every package file and only find source files. You would not report the missing lib files though.

File: cmd/debuginfo.go [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+			missingSource = append(missingSource, pkg.Name)
+		}
+	}
+
+	r.Ok(len(emptyFiles) == 0, "Empty debuginfo files")
```
Comment (matthewrsj) [inline cmd/debuginfo.go]: What is this doing? This isn't actually checking if debuginfo files are present or not. It is checking if the package has any files at all.

File: cmd/debuginfo.go (hunk 2 above)
Comment (Gabib32) [reply cmd/debuginfo.go]: oh I thought I had that, weird - good catch

File: cmd/debuginfo.go (hunk 3 above)
Comment (Gabib32) [reply cmd/debuginfo.go]: Right, that is one of the checks. To get a number of how many debuginfo rpms have no files at all. It's more of a warning than a failure.

Comment (Gabib32) [reply cmd/debuginfo.go]: The package is a debug rpm

Comment (matthewrsj) [reply cmd/debuginfo.go]: ahhhhh ok

File: cmd/debuginfo.go (hunk 1 above)
Comment (matthewrsj) [reply cmd/debuginfo.go]: image

Comment (Gabib32) [reply cmd/debuginfo.go]: HAHAHAHA

**Pull request merged**